ssn/integrated/informativeExamples/InkBird-IBS-TH2.ttl

```diff
@@ -0,0 +1,134 @@
+
+
+<IBSTH2HumiditySensor> rdfs:subClassOf sosa:Sensor ;
+    rdfs:label "Inkbird IBS-TH2 built-in Humidity Sensor"@en ;
+    sosa:observes <airRelativeHumidity> ;
+    ssn-system:hasOperatingRange <IBSTH2HumiditySensorLimits>, <IBSTH2TemperatureSensorLimits> ;
+    sosa:implements <InkBird-IBS-TH2-Procedure> ;
+    ssn-system:hasSystemCapability <IBSTH2HumiditySensorCapability> .
+
+<IBSTH2HumiditySensorAccuracy> a schema:PropertyValue, sosa:Property, ssn-system:Accuracy ;
+    schema:maxValue "0.5" ;
+    schema:minValue "-0.5" ;
+    schema:unitCode unit:Percent ;
+    skos:definition "The accuracy of the humidity sensor is +-0.5 °C in normal temperature and humidity conditions."@en .
+
+<IBSTH2HumiditySensorCapability> a schema:PropertyValue, sosa:Property, ssn-system:SystemCapability ;
+    skos:definition "The capabilities of the humidity sensor in normal temperature and humidity conditions."@en ;
+    ssn-system:hasSystemProperty <IBSTH2HumiditySensorAccuracy> ;
+    ssn-system:inCondition <IBSTH2HumiditySensorNormalLimits>, <IBSTH2TemperatureSensorNormalLimits> .
+
+<IBSTH2HumiditySensorLimits> a ssn-system:OperatingRange ;
+    rdfs:label "Inkbird IBS-TH2 Humidity Sensor Limits"@en ;
+    ssn-system:inCondition <IBSTH2HumiditySensorNormalLimits> .
+
+<IBSTH2HumiditySensorNormalLimits> a schema:PropertyValue, ssn-system:Condition;
+    schema:maxValue "100" ;
+    schema:minValue "0" ;
+    schema:unitCode unit:PERCENT .
+
+<IBSTH2TemperatureSensor> rdfs:subClassOf sosa:Sensor ;
+    rdfs:label "Inkbird IBS-TH2 built-in Temperature Sensor"@en ;
+    sosa:observes <airTemperature> ;
+    ssn-system:hasOperatingRange <IBSTH2HumiditySensorLimits>, <IBSTH2TemperatureSensorLimits> ;
+    sosa:implements <InkBird-IBS-TH2-Procedure> ;
+    ssn-system:hasSystemCapability <IBSTH2TemperatureSensorCapability> .
+
+<IBSTH2TemperatureSensorAccuracy> a schema:PropertyValue, sosa:Property, ssn-system:Accuracy ;
+    schema:maxValue "0.5" ;
+    schema:minValue "-0.5" ;
+    schema:unitCode unit:DEG_C ;
+    skos:definition "The accuracy of the temperature sensor is +-0.5 °C in normal temperature and humidity conditions."@en .
+
+<IBSTH2TemperatureSensorCapability> a schema:PropertyValue, sosa:Property, ssn-system:SystemCapability ;
+    skos:definition "The capabilities of the temperature sensor in normal temperature and humidity conditions."@en ;
+    ssn-system:hasSystemProperty <IBSTH2TemperatureSensorAccuracy> ;
+    ssn-system:inCondition <IBSTH2HumiditySensorNormalLimits>, <IBSTH2TemperatureSensorNormalLimits> .
+
+<IBSTH2TemperatureSensorLimits> a ssn-system:OperatingRange ;
+    rdfs:label "Inkbird IBS-TH2 Temperature Sensor Limits"@en ;
+    ssn-system:inCondition <IBSTH2TemperatureSensorNormalLimits> .
+
+<IBSTH2TemperatureSensorNormalLimits> a schema:PropertyValue, ssn-system:Condition ;
+    schema:maxValue "60" ;
+    schema:minValue "-40" ;
+    schema:unitCode unit:DEG_C .
+
+<InkBird-IBS-TH2-Procedure> a sosa:Procedure ;
```

Comment:
Could this be a SamplingProcedure instead?

Reply:
I am unsure what is the right thing to do here. The document is about the operating characteristics of the entire platform and reports the precision and accuracy of both sensors. 

Common sense implies that the platform is "sampling" the ambient air, but this document is not about that.

Making a note of this for tomorrow's teleconference.